engine/common/mod_bmodel.c

```diff
@@ -207,6 +210,151 @@ static mlumpinfo_t		extlumps[EXTRA_LUMPS] =
 { LUMP_SHADOWMAP, 0, MAX_MAP_LIGHTING / 3, sizeof( byte ), -1, "shadowmap", USE_EXTRAHEADER, (const void **)&srcmodel.shadowdata, &srcmodel.shadowdatasize },
 };
 
+/*
+===============================================================================
+
+			Static helper functions
+
+===============================================================================
+*/
+
+static mip_t* Mod_GetMipTexForTexture( dbspmodel_t* bmod, int textureIndex )
+{
+	if( textureIndex < 0 ||
```

Comment:
You can easily use a shorter variable name like `i` to avoid this expression taking multiple lines. :)